controllers/users.controllers.js

```diff
@@ -0,0 +1,9 @@
+const { selectUsers } = require("../models/users.models")
+
+exports.getUsers = (req, res, next) => {
+    selectUsers().then((users) => {
+        res.status(200).send({ users })
+    }).catch((err) => {
+        next(err)
+    })
+}
```

Comment:
Logic is spot on here, and sending the data back on a key of `users` is correct and appropriate, well done.